cineast-core/src/main/java/org/vitrivr/cineast/core/features/OCRSearch.java

```diff
@@ -17,6 +59,18 @@ public OCRSearch() {
     super(OCR_TABLE_NAME);
   }
 
+  public void processSegment(SegmentContainer shot) {
+    BufferedImage image;
```

Comment:
You don't have to split declaration and initialization, we try to avoid that in the Cineast codebase unless there's a good reason for that